worker/uniter/remotestate/watcher.go

```diff
@@ -208,7 +208,7 @@ func (w *RemoteStateWatcher) loop(unitTag names.UnitTag) (err error) {
 	requiredEvents++
 
 	var seenRelationsChange bool
-	relationsw, err := w.service.WatchRelations()
```

Comment:
Oops, updated. 

I'm not updating the tests to check that the correct unit gets events - it's the server that determines that, so the mock will still behave the same. Or am I missing something?